src/20241106_Multi_GHOAvaxLaunch/AaveV3Arbitrum_GHOAvaxLaunch_20241106.t.sol

```diff
@@ -0,0 +1,500 @@
+  uint64 public constant CCIP_AVAX_CHAIN_SELECTOR = 6433500567565415381;
+
+  event Minted(address indexed sender, address indexed recipient, uint256 amount);
+  event Burned(address indexed sender, uint256 amount);
+  event Transfer(address indexed from, address indexed to, uint256 value);
+
+  function setUp() public {
+    // Execute Avax proposal to deploy Avax token pool
+    vm.createSelectFork(vm.rpcUrl('avalanche'), 53559217);
+
+    // TODO: Decide if we want to deploy this beforehand or in AIP
+    _deployGhoToken();
+
+    // TODO: Remove this deployment once we have deployed pool address
+    _deployCcipTokenPool();
+
+    // TODO: Remove this (will be done on chainlink's side)
+    // Prank chainlink and set up admin role to be accepted on token registry
+    vm.startPrank(AVAX_REGISTRY_ADMIN);
+    TokenAdminRegistry(AVAX_TOKEN_ADMIN_REGISTRY).proposeAdministrator(
+      AVAX_GHO_TOKEN,
+      GovernanceV3Avalanche.EXECUTOR_LVL_1
+    );
+    vm.stopPrank();
+
+    AaveV3Avalanche_GHOAvaxLaunch_20241106 avaxProposal = new AaveV3Avalanche_GHOAvaxLaunch_20241106();
+    GovV3Helpers.executePayload(vm, address(avaxProposal));
+
+    // Switch to Arbitrum and create proposal
+    vm.createSelectFork(vm.rpcUrl('arbitrum'), 279521658);
+
+    // Configure TokenPoolAndProxy for Avalanche
+    // Prank Registry owner
+    vm.startPrank(REGISTRY_ADMIN);
+    _configureCcipTokenPool(TOKEN_POOL_AND_PROXY, CCIP_AVAX_CHAIN_SELECTOR);
+    vm.stopPrank();
+    proposal = new AaveV3Arbitrum_GHOAvaxLaunch_20241106();
+  }
+
+  /**
+   * @dev executes the generic test suite including e2e and config snapshots
+   */
+  function test_defaultProposalExecution() public {
+    defaultTest('AaveV3Arbitrum_GHOAvaxLaunch_20241106', AaveV3Arbitrum.POOL, address(proposal));
+
+    _validateCcipTokenPool();
+  }
+
+  /// @dev Test burn and mint actions, mocking CCIP calls
+  function test_ccipTokenPool() public {
+    GovV3Helpers.executePayload(vm, address(proposal));
+
+    // Mock calls
+    address router = TOKEN_POOL.getRouter();
+    address ramp = makeAddr('ramp');
+    vm.mockCall(
+      router,
+      abi.encodeWithSelector(bytes4(keccak256('getOnRamp(uint64)'))),
+      abi.encode(ramp)
+    );
+    vm.mockCall(
+      router,
+      abi.encodeWithSelector(bytes4(keccak256('isOffRamp(uint64,address)'))),
+      abi.encode(true)
+    );
+
+    // Prank user
+    address user = makeAddr('user');
+
+    // ARB <> ETH
+
+    // Mint
+    uint256 amount = 100e18;
+    uint64 ethChainSelector = proposal.CCIP_ETH_CHAIN_SELECTOR();
+    uint256 startingFacilitatorLevel = _getFacilitatorLevel(address(TOKEN_POOL));
+    uint256 startingGhoBalance = GHO.balanceOf(address(TOKEN_POOL));
+
+    vm.expectEmit(true, true, true, true, address(GHO));
+    emit Transfer(address(0), user, amount);
+
+    vm.expectEmit(false, true, true, true, address(TOKEN_POOL));
+    emit Minted(address(0), user, amount);
+
+    IPoolPriorTo1_5(address(TOKEN_POOL)).releaseOrMint(
```

Comment:
there's no scenario where we would on/offRamp from 1.4 style pools, for even the existing eth<>arb these calls go through the proxyPool which are 1.5/1.5.1 style